applications/ULFapplication/test_examples/bottle_blowing/ProjectParameters.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+domain_size = 3
+
+
```

Comment:
ProjectParameters should be in *.json format

Reply:
Greetings dear @loumalouomega,

Thank you very much for your valuable and appreciated feedback, 

I will be discussing it with Dr. Pavel and do the needed changes accordingly,

Thanks and best regards,

Reply:
Best regards

Reply:
Hi @loumalouomega ! The problemtype used in ULF application is not preparing json files. We can think of updating this, but not within this pull request. Note that in this pull request new functionality is added. We will work on the format in the future.